docs/index.rst

```diff
@@ -81,7 +81,20 @@ Reading data: Statuses
 These functions allow you to get information about single statuses.
 
 .. automethod:: Mastodon.status
+
+Returns a single toot dict for the given status.
+
 .. automethod:: Mastodon.status_context
+
+.. code-block:: python
+    mastodon.status_context(<numerical id>)
+    # Returns
+    {
+     'descendants': A list of toot dicts
+     'ancestors': A list of toot dicts
+    }
+Note that this can only be called on statuses belonging to the currently logged-in user.
```

Comment:
That's not correct, status context is available to any user. It doesn't even need a user context (aka an app without an authorized user could access it)